charts/node/templates/node-statefulset.yaml

```
@@ -41,7 +41,60 @@ spec:
             - name: data
               mountPath: {{ .Values.common.basePath }}
         {{- end }}
-        {{- if eq .Values.common.role "validator" }}
+        {{- if .Values.node.chainData.cleanup }}
+        - name: cleanup-chain
+          image: curlimages/curl:latest
+          command:
+            - /bin/sh
+            - -c
+            - |
+              rm -rvf {{ .Values.common.basePath }}/chains
```

```suggestion
              rm -rvf {{ .Values.common.basePath }}/{{ include "chainflip-node.databasePath" . }}/paritydb
              rm -rvf {{ .Values.common.basePath }}/{{ include "chainflip-node.databasePath" . }}/db
```

Otherwise we will yeet session keys.